.gitignore

```diff
@@ -9,3 +9,5 @@ deps/downloads
 deps/src/hwloc-*
 deps/usr
 deps/build.log
+src/Hwloc.jl
+Manifest.toml
```

Comment:
It's best to not commit the manifest, as the manifest is specific to a particular Julia version. Committing it creates problems for CI, and for people who use a different Julia version than the one that generated the manifest.

Reply:
I fully agree. However, I'm not committing the `Manifest.toml` here but, in fact, am adding it to the `.gitignore` for precisely this reason: I don't want to accidentally commit it.

(See https://github.com/crstnbr/Hwloc.jl/tree/memorychildren, no Manifest.toml there.)

Reply:
D'oh. That tells you how to interpret my other comments.